projects/plugins/boost/app/data-sync/Image_Size_Analysis_Ignored_Images.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Automattic\Jetpack_Boost\Data_Sync;
+
+use Automattic\Jetpack\WP_JS_Data_Sync\Contracts\Entry_Can_Get;
+use Automattic\Jetpack\WP_JS_Data_Sync\Contracts\Entry_Can_Set;
+use Automattic\Jetpack\WP_JS_Data_Sync\Data_Sync_Option;
+
+class Image_Size_Analysis_Ignored_Images implements Entry_Can_Get, Entry_Can_Set {
```

Comment:
I guess the reason behind this custom Entry is to avoid `delete`? In that case, might be useful to add some comment explaining that and why we don't want `delete` there.

Reply:
This is a WIP. It's going to become a lazy entry too eventually - because the ignored images data lives on wpcom. For now it's just implementing the bare minimum to make it work.